modin/core/storage_formats/pandas/utils.py

```diff
@@ -65,6 +65,9 @@ def compute_chunksize(
         If axis is 1 or 0, returns an integer number of rows/columns to split the
         DataFrame. If axis is None, returns a tuple containing both.
     """
+    default_block_size = (
+        default_block_size if default_block_size else MinElementsInPartition.get()
```

Comment:
either use `if default_block_size is None` or just simply use `default_block_size = default_block_size or MinElementsInPartition.get()` if `default_block_size=0` should be replaced, too

Reply:
I switched to `if default_block_size is None`